src/main/java/frc/robot/hardware/rev/motors/SparkMaxWrapper.java

```diff
@@ -0,0 +1,22 @@
+package frc.robot.hardware.rev.motors;
+
+import com.revrobotics.CANSparkMax;
+import edu.wpi.first.math.geometry.Rotation2d;
+import frc.utils.Conversions;
+
+public class SparkMaxWrapper extends CANSparkMax {
+
+	public SparkMaxWrapper(SparkMaxDeviceID deviceID) {
+		super(deviceID.id(), deviceID.type());
+		super.restoreFactoryDefaults();
```

Comment:
This seems dangerous, why are we doing this?

Reply:
when i create my motor, i dont want it to have saved configurations. i want it to have default one and configure it on run with what i need